deta/base.py

```diff
@@ -116,22 +114,16 @@ def insert(
         expire_in: int = None,
         expire_at: typing.Union[int, float, datetime.datetime] = None,
     ):
-        if not isinstance(data, dict):
-            data = {"value": data}
-        else:
-            data = data.copy()
-
+        data = data.copy() if isinstance(data, dict) else {"value": data}
         if key:
             data["key"] = key
 
         insert_ttl(data, self.__ttl_attribute, expire_in=expire_in, expire_at=expire_at)
-        code, res = self._request(
-            "/items", "POST", {"item": data}, content_type=JSON_MIME
-        )
+        code, res = self._request("/items", "POST", {"item": data}, content_type=JSON_MIME)
         if code == 201:
             return res
         elif code == 409:
-            raise Exception("Item with key '{4}' already exists".format(key))
+            raise ValueError("Item with key '{4}' already exists".format(key))
```

Comment:
I think this one should be changed into `f-string` too  
```suggestion
            raise ValueError(f"Item with key '{key}' already exists")
```